handsontable/src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -560,6 +566,24 @@ export class ManualColumnResize extends BasePlugin {
     }
   }
 
+  /**
+   * Callback for "contextmenu" event triggered on element showing move handle. It removes handle and guide elements.
```

Comment:
Please mark the method with the `@private` JSDoc tag. Otherwise, the method will be generated in Docs, and I guess it shouldn't.

Reply:
Done within https://github.com/handsontable/handsontable/pull/10416/commits/b7ca869936281bac45ddc7dbb33da23193f5b022.

Reply:
Thanks! Can I ask one more time for a fix? There is a typo `this.isTriggeredbyRMB` should be `this.isTriggeredByRMB`.